gha_clone_releases/main.py

```diff
@@ -12,6 +12,7 @@
 ###START_INPUT_AUTOMATION###
 INPUTS = {
     "token": {"description": "Github token", "required": True},
+    "dest_token": {"description": "Github token for destination repo", "required": False},
```

Comment:
Remove this line and add your new input to action.yaml, then run make generate-inputs (or run pre-commit). This is generated automatically from action.yaml.

Reply:
I'm not familiar with python pre-commit, and i don't have python environment in my computer. So do the job with hand, it's okay for you ?